doc/MAIN.md

```diff
@@ -1,5 +1,17 @@
+# MapLibreGlDirections
+
+For the sakes of your convenience, make sure you've enabled the "Public" filter:
+
+![Enabling the "Public" filter](https://raw.githubusercontent.com/maplibre/maplibre-gl-directions/main/doc/images/public-filter.png)
+
+---
```

Comment:
I don't understand this part.

Reply:
Oh. It's an image that tells how the reader can enable the filter to only show public exports. By default (and you can't change that) everything is shown in the docs, and that's quite inconvenient. So nothing super-special about it, just a tip for the one who reads the docs.